registry-manager-db-api/src/main/java/de/adorsys/registry/manager/repository/AspspRepository.java

```diff
@@ -16,20 +16,28 @@
     AspspPO save(AspspPO aspsp);
 
     /**
-     * Saves a list of aspsps into existing Lucene indexes.
+     * Saves a list of aspsps into existing database.
      * <p>
-     * Writes all Aspsp objects into the current Lucene repository. The iteration through input
+     * Writes all Aspsp objects into the current repository. The iteration through input
      * collection of aspsps is taken place, each object is either added or updated, if it
-     * already exists within the repository.
+     * already exists within the repository. New random id is generated if input entity
+     * id is NULL.
      *
-     * @param aspsps a list of aspsp objects to be added into the repository
+     * @param aspsps, a list of aspsp objects to be added into the repository
      */
     void saveAll(List<AspspPO> aspsps);
 
     void deleteById(UUID aspspId);
 
     /**
-     * Deletes all records from the existing Lucene indexes.
+     * Deletes all records from the existing database.
      */
     void deleteAll();
+
+    /**
+     * Deletes a list of {@link AspspPO} from the existing database.
+     *
+     * @param aspsps, a list of objects
+     */
+    void deleteAll(List<AspspPO> aspsps);
```

Comment:
Maybe we'd better call this method as simply `delete()` instead of `deleteAll()` to be more clear, what do you think?

Reply:
hmm... I wouldn't agree with you here. Calling just *delete* sounds too abstract for me. It could be inferred like deleting one from a list, all elements, any item from a list, etc. You can't be sure without checking the implementation.

I'd keep the naming like it is if you mind.